tests/test_download_conda_packages.py

```diff
@@ -0,0 +1,144 @@
+from pathlib import Path
+
+import pytest
+from ruamel.yaml import YAML
+
+from ib_manifest_util import PACKAGE_DIR, TEST_DATA_DIR
+from ib_manifest_util.download_conda_packages import download_packages
+
+small_package_url_and_filename = [
+    "https://conda.anaconda.org/conda-forge/noarch/backports-1.0-py_2.tar.bz2",
+    "backports-1.0-py_2.tar.bz2",
+]
+
+
+def load_yaml_for_tests(file_path: str | Path, loader_type: str = "safe") -> dict:
+    """Load a yaml file.
+
+    This provides a method for loading yaml files independent of utils.load_yaml.
+
+    Args:
+        file_path: Path to yaml file.
+        loader_type: Type of loader to use (see ruamel.yaml).
+
+    Returns: dict
+    """
+    yaml_loader = YAML(typ=loader_type)
+    file_path = Path(file_path).resolve()
+    with open(file_path, "r") as f:
+        return yaml_loader.load(f)
+
+
+def assert_file_exists_and_has_data_then_delete(
```

Comment:
```suggestion
def assert_exist_then_delete(
```
I appreciate the verbosity, but I think its a little much